test/models/accountDelete.ts

```diff
@@ -34,7 +38,10 @@ describe('AccountDelete Transaction Verification', function () {
         } as any
 
         assert.throws(
-            () => verifyAccountDelete(invalidDestination),
+            () => {
+                verifyAccountDelete(invalidDestination)
+                verify(invalidDestination)
+            },
             ValidationError,
             "AccountDelete: missing field Destination"
         )
```

Comment:
Each of the `assert.throws` needs a separate `assert.throws` for `verify[SpecificTransaciton]` vs `verify` - `assert.throws` only checks if an error is thrown somewhere in the block, not that each throws an error